pkg/systemd/hwinfo.jsx

```diff
@@ -268,6 +269,7 @@ class HardwareInfo extends React.Component {
         if (this.props.info.memory.length > 0) {
             memory = (
                 <ListingTable caption={ _("Memory") }
+                    rowKey={({ rowData, rowIndex }) => rowData.id.title}
```

Comment:
Same here, probably needs to use `rowData[_("ID").toLowerCase()]` or something similarly ugly